### PR TITLE
added code system selector reusable component

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-5727](https://jira.cms.gov/browse/MAT-5727)
(Optional) Related Tickets:

### Summary
added code system selector reusable component
### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

**Comment by RohitKandimalla** [comment]
Can you provide a screenshot of the implementation ?
Recently we talked about adding the UI screenshots whenever possible for Code Reviews.

File: tailwind.config.js [hunk 1]
```diff
@@ -133,6 +133,9 @@ module.exports = {
         96: "24rem",
         128: "32rem",
       },
+      width: {
+        64: "19rem",
```
Comment (RohitKandimalla) [inline tailwind.config.js]: Why are writing custom width, Can't we take the closet available in Tailwind ?

File: src/components/common/dateTimeInterval/DateTimeInterval.tsx [hunk 2]
```diff
@@ -2,29 +2,46 @@ import React from "react";
...
   endDateTime,
   handleEndDateTimeChange,
+  canEdit,
 }: DateTimeIntervalProps) => {
   return (
     <div>
+      <h5 style={dateTimeTextFieldStyle}>Time Range</h5>
       <FormControl
-        style={{ display: "flex", gap: "40px", flexDirection: "row" }}
+        style={{ display: "flex", gap: "32px", flexDirection: "row" }}
```
Comment (RohitKandimalla) [inline src/components/common/dateTimeInterval/DateTimeInterval.tsx]: Can we use tailwind here? the only reason is that Tailwind Classes are responsive and will reduce overhead in the future

Comment (sb-prateekkeerthi) [reply src/components/common/dateTimeInterval/DateTimeInterval.tsx]: I tried with tailwind but I didn't workout. I used tw="flex flex-row gap-8" in the form control but it is not placing the divs side by side

**Comment by sb-prateekkeerthi** [comment]
> Can you provide a screenshot of the implementation ? Recently we talked about adding the UI screenshots whenever possible for Code Reviews.

Here is the picture of the implementation

Screen Shot 2023-06-21 at 2 17 12 PM

File: tailwind.config.js (hunk 1 above)
Comment (sb-prateekkeerthi) [reply tailwind.config.js]: I just removed it thank you

File: src/components/common/dateTimeInterval/DateTimeInterval.tsx (hunk 2 above)
Comment (sb-prateekkeerthi) [reply src/components/common/dateTimeInterval/DateTimeInterval.tsx]: fixed it, thanks for the help

**Review by RohitKandimalla — DISMISSED** [review]

**Review by RohitKandimalla — DISMISSED** [review]

**Review by adongare — APPROVED** [review]

**Pull request merged**